tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -485,7 +485,11 @@ private function preventStaleDbVersionError(array $operations, Context $context)
             switch ($operation->name) {
                 case 'distinct':
                     $hasDistinct = true;
-                    $collection = $context->getEntityMap()[$operation->object];
+                    /* TODO: If this operation references an entity that would
+                     * be created by a createEntities test runner operation, the
+                     * assertion below will fail; however, there is no need to
+                     * address this until such a transaction test is created. */
+                    $collection = $context->getEntityMap()[$operation->object] ?? null;
```

Comment:
Working around this would require running the `createEntities` operation ahead of time, which is a can of worms.